core/services/validators.go

```diff
@@ -128,12 +128,12 @@ func validateFluxMonitor(i models.Initiator, j models.JobSpec, store *store.Stor
 	if i.Address == utils.ZeroAddress {
 		fe.Add("no address")
 	}
+	if len(i.ValueTriggers) == 0 {
+		fe.Add("must have at least one value trigger")
```

Comment:
I believe we want to support only idleThreshold. @thodges-gh or @spooktheducks ?

Reply:
I will add a trivial value trigger, which always triggers. I would rather require the user be explicit about that than make it the default.